build.sh

```diff
@@ -31,9 +36,32 @@ _notify_github() {
     $REPO_ROOT_DIR/tools/github_update.py $1 build:sdk $2
 }
 
+merge_master() {
+    echo "attmpting to merge changes from master"
+    # git sucks and won't let you update files without knowing a name
+    echo "Creating fake user."
+    command="git config --global user.email pullrequestbot@mesospherebot.com"
```

Comment:
FWIW the `--global` may not be necessary, see:
- docs generator script which pushes to gh-pages: https://github.com/mesosphere/dcos-commons/blob/master/docs/generate.sh#L111
- sample result: https://github.com/mesosphere/dcos-commons/commit/0ee0f3a22a731a4ae3a035b7a56e3b873f0b381c